features/layout/page-container/page-container.module.scss

```diff
@@ -19,13 +19,15 @@
 }
 
 .contentContainer {
-  min-height: calc(100vh - 2 * space.$s8 - misc.$header-height);
+  min-height: calc(
+    100vh - 2 * space.$s8 - misc.$header-height - misc.$footer-height
+  );
```

Comment:
I took a slightly different yet more complex to refactor approach that involves `display: grid`. You can check the solution video for that